solr/core/src/java/org/apache/solr/handler/component/FacetComponent.java

```diff
@@ -1033,9 +1033,9 @@ private void refinePivotFacets(ResponseBuilder rb, ShardRequest sreq) {
       }
       
       for (Entry<String,List<NamedList<Object>>> pivotFacetResponseFromShard : pivotFacetResponsesFromShard) {
-        PivotFacet masterPivotFacet = fi.pivotFacets.get(pivotFacetResponseFromShard.getKey());
-        masterPivotFacet.mergeResponseFromShard(shardNumber, rb, pivotFacetResponseFromShard.getValue());  
-        masterPivotFacet.removeAllRefinementsForShard(shardNumber);
+        PivotFacet primaryPivotFacet = fi.pivotFacets.get(pivotFacetResponseFromShard.getKey());
+        primaryPivotFacet.mergeResponseFromShard(shardNumber, rb, pivotFacetResponseFromShard.getValue());
+        primaryPivotFacet.removeAllRefinementsForShard(shardNumber);
```

Comment:
This has nothing to do with master/slave replication. Is this something we want to change too? If so, I guess this is an aggregation?